src/pages/landing/Landing.tsx

```diff
@@ -0,0 +1,201 @@
+  <Box className="flex flex-row-reverse flex-1">
+    <Box className="flex flex-col justify-center m-0 min-w-[50vw] bg-white">
+      <Box className="text-center">
+        <Typography variant="h3" className="font-bold mb-4 tracking-[.2em]">
+          Testing Tool
+        </Typography>
+        <Typography variant="h6" className="font-normal mb-40 text-[#A6A7AD]">
+          Connect your in-browser wallet to login/sign-up
+        </Typography>
+        <Button variant="contained" size="large" className="py-3 px-4 normal-case font-medium" onClick={props.onSubmit}>
+          Connect your wallet
+        </Button>
+        <ConnectWallet />
+      </Box>
+    </Box>
+  </Box>
+);
+
+const SubscriptionSection = (props: { tiers: Tier[], tierId: string|null, onSelect: (tierId: string) => void }) => (
+  <Box className="flex flex-col m-0 pt-24 min-w-[50vw] bg-white">
+    <Box className="text-center">
+      <Typography variant="h4" className="font-bold mb-2 tracking-[.2em]">
+        Testing Tool
+      </Typography>
+      <Typography variant="subtitle1" className="font-normal mb-16 text-[#A6A7AD]">
+        Please choose the Package you wish to subscribe to.
+      </Typography>
+      <Box className="flex flex-row px-2">
+        {props.tiers.map(tier => (
+          <div key={tier.id} className={props.tierId === tier.id ? 'package-type-active' : 'package-type'}>
+            <Typography variant="h5" className="font-bold my-8">
+              {tier.name}<br/>Premium Tier
+            </Typography>
+            <Typography variant="h2" className="font-normal mb-24">
+              ${tier.usdPrice.toFixed(2)}
+            </Typography>
+            <Typography variant="body1" className="font-bold">
+              {tier.subtitle}
+            </Typography>
+            <Typography variant="body1" className="font-normal mb-12">
+              {tier.features.map(feature => (
+                <span key={feature.name}>{feature.name}<br/></span>
+              ))}
+            </Typography>
+            <Button
+              variant="contained"
+              size="large"
+              className="py-3 px-14"
+              color={!props.tierId || props.tierId === tier.id ? 'info' : 'primary'}
+              onClick={() => props.onSelect(tier.id)}
+            >
+              {props.tierId === tier.id ? 'Selected' : 'Select'}
+            </Button>
+          </div>
+        ))}
+      </Box>
+    </Box>
+  </Box>
+);
+
+const RegisterSection = (props: { tierId: string, count: number, price: number, onSubmit: (event: React.SyntheticEvent) => void }) => (
+  <Box className="flex flex-col m-0 pt-24 min-w-[50vw] bg-white text-center">
+    <Typography variant="h4" className="font-bold mb-2 tracking-[.2em] capitalize">
+      Auditor profile
+    </Typography>
+    <Typography variant="subtitle1" className="font-normal mb-16 text-[#A6A7AD]">
+      Please complete the information to create your account.
```

Comment:
```suggestion
      Please complete your user profile information
```

Same as previous comment.